src/main/webapp/js/omniComment.js

```diff
@@ -223,19 +223,19 @@ $(document).ready(function() {
         // if not all list elements are hidden within fbResponse, then show fbResponse
         if ($(comment).prop('class').toString().includes(classNameForCommentsInFeedbackResponse)) {
             if ($(comment).parent().find('li[style*="display: none"]').length !== $(comment).parent().find('li').length) {
-                var commentListRegionForFeedbackResponse = $(comment).parent().parent().parent();
+                var commentListRegionForFeedbackResponse = $(comment).closest('tr');
```

Comment:
I will create an issue for this, I was tempted to fix the `.prev()`s in this PR :stuck_out_tongue: